core/search_index.py

```
@@ -0,0 +1,333 @@
+                        if not self._is_stale(conn, customer_path, prefix):
+                            continue
+
+                        _emit(f"Indexing {customer}…")
+                        conn.execute(
+                            "DELETE FROM jobs WHERE customer=? AND prefix=?",
+                            (customer, prefix),
+                        )
+
+                        try:
+                            jobs = app_context.find_job_folders(customer_path)
+                        except Exception as exc:
```

Ruff (0.15.12)

[warning] Do not catch blind exception: `Exception`

(BLE001)